var/spack/repos/builtin/packages/uftrace/package.py

```diff
@@ -23,11 +23,10 @@ class Uftrace(AutotoolsPackage):
 
     # The build process uses 'git describe --tags' to get the package version
     version("master", branch="master", get_full_repo=True)
-    version("0.11", sha256="101dbb13cb3320ee76525ec26426f2aa1de4e3ee5af74f79cb403ae4d2c6c871")
-    version("0.10", sha256="b8b56d540ea95c3eafe56440d6a998e0a140d53ca2584916b6ca82702795bbd9")
```

Comment:
If there are issues with these versions then it would be better to add comments explaining why they should not be included so no-one else comes along and adds them later.